modules/actions/add_midpoint.js

```diff
@@ -1,5 +1,5 @@
-import { geomEdgeEqual } from '@id-sdk/geom';
-import { utilArrayIntersection } from '../util';
+import { geomEdgeEqual } from '@id-sdk/math';
```

Comment:
ha, thanks for fixing this /geom -> /math import statement.

Reply:
Yea I've been trying to consolidate them as I go.. It's not bad to do it either way, but I'm a big fan of fewer lines of code.